### ISSUE TITLE
Live photo cannot be generated from local video

### ISSUE THREAD

**Issue by onepiececrush** [YuheiNakasaka/flutter_live_photos#1]
Live photo is not supported for local video generation

**Comment by YuheiNakasaka** [comment]
@cailei1 
I implemented the feature. You can use like below.
```dart
// Local File
LivePhotos.generate(
  localPath: "<local mp4 file path>"
).then((status) {
  if (status) {
    print("Success to generate LivePhotos");
  } else {
    print("Failed to convert");
  }
});
```

More detailed example is [here](https://pub.dev/packages/live_photos/example).

It's available in v0.3.0.
[https://pub.dev/packages/live_photos](https://pub.dev/packages/live_photos)

**Comment by onepiececrush** [comment]
Thanks very much!    I've tried this method, but the application just crashes. my local path value:"/var/mobile/Containers/Data/Application/AD2C51D8-E176-4E99-9C45-BCB60B6B51CD/Documents/Trimmer/trim_trimmed:Dec3,2020-23:51:54.mp4"
